bootstrap/018_warehouse_updates.sql

```diff
@@ -34,7 +33,7 @@ begin
     return object_construct('migrate1', migrate1, 'migrate2', migrate2);
 end;
 
-CREATE OR REPLACE PROCEDURE internal.refresh_warehouse_events(migrate boolean) RETURNS STRING LANGUAGE SQL
+CREATE OR REPLACE PROCEDURE internal.refresh_warehouse_events(migrate boolean, input variant) RETURNS OBJECT LANGUAGE SQL
```

Comment:
given that variant is input, we should declare (via comment or similar) what the expected fields must be/could be.

Reply:
> given that variant is input

The previous code was doing cast to `VARIANT` for no apparent reason. I unwound some of that (after I missed updating a `RETURNS`). I'll just swap them all to be OBJECT since that's what is intended.

Reply:
I'm speaking to the fields in the object/variant not the type itself. 

It used to be that the object was entirely inside the procedure. You've moved it to the signature which is why I'm suggesting we document the expected structure.

Reply:
Ah, I see. Makes sense